include/eve/function/expx2.hpp

```diff
@@ -12,6 +12,68 @@
 
 namespace eve
 {
+  //================================================================================================
+  //! @addtogroup exponential
+  //! @{
+  //! @var expx2
+  //!
+  //! @brief Callable object computing \f$e^{\pm x^2}\f$.
+  //!
+  //! **Required header:** `#include <eve/function/expx2.hpp>`
+  //!
+  //! #### Members Functions
+  //!
+  //! | Member       | Effect                                                     |
+  //! |:-------------|:-----------------------------------------------------------|
+  //! | `operator()` | the  computation of expx2                                  |
```

Comment:
This should be a sentence with a verb no ?

Reply:
done

Reply:
done